content/docs/csidriver/features/powerflex.md

```diff
@@ -861,6 +861,16 @@ enableQuota: false
 ...
 ```
 
+## Volume Size and Rounding Rules
+For NFS File Shares, the minimum supported volume size is 3 GiB. If request size is smaller than 3 GiB, the volume will be automatically provisioned as 3 GiB.
```

Comment:
How about Gi in place of GiB? As Using Gi instead of GiB is common in Kubernetes YAML files. While GiB may be easier to understand at first glance because it explicitly refers to gibibytes.

Reply:
Yes, it must be `Gi` as in the code it is `Gi`.